docs/src/extend/custom-rules.md

```diff
@@ -117,9 +117,9 @@ The `context` object contains additional functionality that is helpful for rules
 
 Additionally, the `context` object has the following methods:
 
-* `getAncestors()` - returns an array of the ancestors of the currently-traversed node, starting at the root of the AST and continuing through the direct parent of the current node. This array does not include the currently-traversed node itself.
+* `getAncestors()` - (**Deprecated:** Use `SourceCode.getAncestors(node)` instead.) returns an array of the ancestors of the currently-traversed node, starting at the root of the AST and continuing through the direct parent of the current node. This array does not include the currently-traversed node itself.
```

Comment:
Should we also document the new methods `SourceCode#getAncestors(node)` and `SourceCode#getDeclaredVariables(node)` somewhere in this document?

Reply:
We don't already have a section outlining methods on `SourceCode`, so there is no natural place to insert these right now. We can always do another pass of docs later if we want to update with that info.